Modix/Modules/UserInfoModule.cs

```diff
@@ -7,18 +7,22 @@
 using Discord.Commands;
 using Humanizer;
 using Humanizer.Localisation;
+using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Abstractions;
+using Modix.Data;
 
 namespace Modix.Modules
 {
     public class UserInfoModule : ModuleBase
     {
-        public UserInfoModule(ILogger<UserInfoModule> logger)
+        public UserInfoModule(ModixContext modixContext, ILogger<UserInfoModule> logger)
         {
+            Db = modixContext;
```

Comment:
As mentioned above, you need a repository here, not `ModixContext` directly.

Reply:
Responded to the comment above.